### ISSUE TITLE
Problems with the template 'Striped'

### ISSUE THREAD

**Issue by MegaBedder** [dignajar/bludit-themes#12]
When you upload the theme folder 'Striped' generates the following error.

in /admin/themes

```
Notice: Undefined index: name in G:\wamp\bludit\kernel\admin\views\themes.php on line 23
Notice: Undefined index: description in G:\wamp\bludit\kernel\admin\views\themes.php on line 34
Notice: Undefined index: version in G:\wamp\bludit\kernel\admin\views\themes.php on line 35
Notice: Undefined index: website in G:\wamp\bludit\kernel\admin\views\themes.php on line 36
Notice: Undefined index: author in G:\wamp\bludit\kernel\admin\views\themes.php on line 36
```

in index:

```
Notice: Undefined index: theme-data in G:\wamp\bludit\kernel\boot\rules\99.themes.php on line 43
```

in Bludit v0.6.2 "Jake" and Github version

**Comment by Dipchikov** [comment]
I fix this error!  A stupid mistake:)

**Comment by MegaBedder** [comment]
Fix #13, good job @Dipchikov :+1:
